doc-content/enterprise-only/getting-started-kogito/con-migration-to-kogito-overview.adoc

```diff
@@ -0,0 +1,9 @@
+[id="con-migration-to-kogito-overview_{context}"]
+= Overview of migration to {KOGITO} microservices
+
+[role="_abstract"]
+Migration to {KOGITO} microservices, describes the process of migrating the existing services that you developed in {CENTRAL} to {KOGITO} microservices. You can migrate the decision services artifacts, including DMN, PMML, and DRL to {KOGITO} deployments.
+
+The decision services that are implemented using DMN models in {CENTRAL} can be migrated by moving the DMN resources from KJAR artifacts to {KOGITO} archetypes. Similarly, you can migrate the prediction services that are implemented using PMML models by moving the PMML resources from KJAR artifacts to {KOGITO} archetype.
+
+Also, the decision services that are implemented using DRL can be migrated to {KOGITO} deployments using {QUARKUS} REST endpoint. Migrating the DRL files using {QUARKUS} enables you to leverage the {QUARKUS} features while embracing the programming model of {KOGITO}, which is based on the rule units.
```

Comment:
I'd replace this sentence with something like this
`Also, the decision services that are implemented using DRL can be simply wrapped with {QUARKUS} REST endpoint. However bringing them to {KOGITO} enables you to leverage the main {QUARKUS} features like hot reload and native compilation, while fully embracing the programming model of {KOGITO}, which is based on the rule units, will also enable the automatic generation of those REST endpoints.`
That said I also find that translating the `{QUARKUS}` placeholder with `Red Hat build of Quarkus` (and same thing for Kogito) makes sentences like this extremely hard to read.

Reply:
Done, changing the paragraph as follows:

Also, the decision services that are implemented using DRL can be enclosed with {QUARKUS} REST endpoint. However, migrating the same decision services to {KOGITO} enables you to use major Quarkus features, such as hot reload and native compilation. Using major Quarkus features and embracing the programming model of Kogito, enables the automatic generation of the Quarkus REST endpoints.